src/backend/Error.h

```diff
@@ -36,7 +36,15 @@ namespace backend {
     //
     // Returning an error is done via:
     //   NXT_RETURN_ERROR("My error message");
-#define NXT_RETURN_ERROR(EXPR) return MakeError(EXPR, __FILE__, __func__, __LINE__)
+#define NXT_RETURN_ERROR(MESSAGE) return MakeError(MESSAGE, __FILE__, __func__, __LINE__)
+#define NXT_TRY_ASSERT(EXPR, MESSAGE)                             \
+    {                                                             \
+        if (!(EXPR)) {                                            \
+            NXT_RETURN_ERROR(MESSAGE);                            \
+        }                                                         \
+    }                                                             \
+    for (;;)                                                      \
```

Comment:
Only `break` is required I think.

Reply:
As opposed to `for (;;) break`? Doesn't seem to work. I copied this from NXT_TRY.